metaci/users/handlers.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth.models import Group
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from guardian.conf import settings as guardian_settings
+from metaci.users.models import User
+
+@receiver(post_save, sender=User)
+def add_public_group(sender, **kwargs):
+    user = kwargs['instance']
+    created = kwargs['created']
+    if not created:
+        return
+
+    if user.username == guardian_settings.ANONYMOUS_USER_NAME:
+        return
+
+    try:
+        user.groups.get(name='Public')
```

Comment:
is there a scenario where in the initial post_save, the user already has the group assigned? this seems extraneous, since we've already short circuited if this wasn't the save where the user was created.